src/rpc/blockchain.cpp

```diff
@@ -786,6 +786,49 @@ static RPCHelpMan getmempoolentry()
     };
 }
 
+static RPCHelpMan getmempoolspendingtx()
```

Comment:
Sorry to nitpick but I was confused in the beginning by the name as well (I thought you wanted the child of a mempool transaction). Maybe `gettxspendingprevout` would be better, but I'm sure there's someone out there who would find it even more confusing :sweat_smile:

Reply:
I agree, I'm not a fan of the name! Should it be `getmempooltxspendingprevout` though to highlight that we return a mempool tx (with ancestor / descendant feerates, which are useful in fee-bumping cases)? It's a long name, but at least it's quite obvious what it does (hopefully).

Reply:
I updated the name, but now that is accepts multiple prevouts, I'm not sure whether I should make it plural or not...
None of the names I find are completely satisfying, so I'll let reviewers chime in with better proposals!